packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -82,7 +83,12 @@ class StyletronClient implements StandardEngine {
     const onNewStyle = (cache, id, value) => {
       const {pseudo, block} = value;
       const sheet: CSSStyleSheet = (this.styleElements[cache.key].sheet: any);
-      const rule = styleBlockToRule(atomicSelector(id, pseudo), block);
+      let rule;
+      if (pseudo.includes("@supports")) {
```

Comment:
Should be `@supports` treated exactly same as `:hover` and other pseudo classes? If so, could this condition be 
moved here: https://github.com/styletron/styletron/blob/master/packages/styletron-engine-atomic/src/inject-style-prefixed.js#L67 ?

Reply:
Yep 👍 Good spot!

I think it may be the cleanest way without unintended side effects. I think `media` gets handled a little differently with reordering etc.